docs/index.rst

```diff
@@ -4,29 +4,34 @@ Read the Docs: Documentation Simplified
 .. meta::
    :description lang=en: Automate building, versioning, and hosting of your technical documentation continuously on Read the Docs.
 
-
 `Read the Docs`_ simplifies software documentation
-by automating building, versioning, and hosting of your docs for you.
+by building, versioning, and hosting of your docs, automatically.
 Think of it as *Continuous Documentation*.
 
-Never out of sync
+Never out of sync |:arrows_counterclockwise:|
     Whenever you push code to your favorite version control system,
     whether that is Git, Mercurial, Bazaar, or Subversion,
     Read the Docs will automatically build your docs
     so your code and documentation are always up-to-date.
+    Read more about :doc:`/webhooks`.
 
-Multiple versions
+Multiple versions |:card_index_dividers:|
     Read the Docs can host and build multiple versions of your docs
     so having a 1.0 version of your docs and a 2.0 version
     of your docs is as easy as having a separate branch or tag in your version control system.
+    Read more about :doc:`/versions`.
 
-Free and open source
-    Read the Docs is free and open source and hosts documentation
-    for nearly 100,000 large and small open source projects
+Open Source and User Focused |:heartbeat:|
+    Our code is free and `open source <https://github.com/readthedocs/>`_.
+    :doc:`Our company </about>` is boostrapped and 100% user focused.
```

Comment:
```suggestion
    :doc:`Our company </about>` is bootstrapped and 100% user focused.
```
I'm not sure of a better way to say bootstrapped here, but it feels a bit unnecessary in this context, and "100% user focused" is maybe too fluffy of a statement.

What thi is